.vscode/launch.json

```diff
@@ -40,7 +42,7 @@
 			"port": 6010,
 			"sourceMaps": true,
 			"outFiles": [
-				"${workspaceRoot}/client/server/**/*.js",
+				"${workspaceRoot}/server/out/**/*.js",

```

Comment:
We must check that any code modification in client and server is properly recompiled before all debug launch configurations.

Reply:
`npm run compile` at the root used to recompile both client and server. Is it still the case?

Reply:
It looks like it's still the case, but much faster! great job!